osquery/tables/system/python_packages.cpp

```diff
@@ -153,10 +209,18 @@ QueryData genPythonPackagesImpl(QueryContext& context, Logger& logger) {
     }
   }
   for (const auto& key : paths) {
-    genSiteDirectories(key, results, logger);
+    genSiteDirectories(key, results, logger, 0);
   }
 
   if (isPlatform(PlatformType::TYPE_OSX)) {
```

Comment:
We should also support listing user directories on Windows devices.

PS: If we cannot support it (for whatever reason), we should document it in the table specs.

Reply:
From the sounds of this comment it appears that case is handled:
https://github.com/osquery/osquery/blob/master/osquery/tables/system/python_packages.cpp#L183-L185

However, I am not very familiar with python on windows so sounds like a good idea to download a vm and double check.